index.html

```diff
@@ -1,75 +1,108 @@
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+		<div class="container">
+			<section class="top-content">
+				<div class="jumbo">
+					<img src="/img/jumbo.jpg" alt="" />
+				</div>
+				<div class="text-container">
+					<h2>The Future</h2>
+					<p>
+						Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+						Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+						imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+						egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+					</p>
+				</div>
+				<div class="text-container">
+					<h2>The Past</h2>
+					<p>
+						Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+						Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+						imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+						egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+					</p>
+				</div>
+			</section>
 
-        <section class="bottom-content">
+			<section class="middle-content">
+				<h2>Why Did It Have To Be Boxes...</h2>
 
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
+				<div class="boxes">
+					<div class="box box1">Box 1</div>
+					<div class="box box2">Box 2</div>
+					<div class="box box3">Box 3</div>
+					<div class="box box4">Box 4</div>
+					<div class="box box5">Box 5</div>
+					<div class="box box6">Box 6</div>
+					<div class="box box7">Box 7</div>
+					<div class="box box8">Box 8</div>
+					<div class="box box9">Box 9</div>
+					<div class="box box10">Box 10</div>
```

Comment:
Amazing job, another way to change the box color would be using the  :nth-child( ) selector in CSS ->https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child